mmf/utils/checkpoint.py

```diff
@@ -379,11 +384,15 @@ def _get_vcs_fields(self):
             "git/diff": self.git_repo.git.diff("--no-prefix"),
         }
 
+    def save_func(self, *args):
+        return xm.save(*args) if is_xla() else torch.save(*args)
+
     def save(self, update, iteration=None, update_best=False):
         # Only save in main process
-        if not is_master():
+        if not is_master() and not is_xla():
             return
 
+        logger.info("Checkpoint save operation started!")
```

Comment:
I believe this is printed somewhere else as well. No need to repeat.

Reply:
I think forgot to remove it :(, will push this change with the next feedback.